oauthproxy.go

```diff
@@ -1248,3 +1251,27 @@ func (p *OAuthProxy) errorJSON(rw http.ResponseWriter, code int) {
 	// application/json
 	rw.Write([]byte("{}"))
 }
+
+// LoggingCSRFCookiesInOAuthCallback Log all CSRF cookies found in HTTP request OAuth callback,
+// which were successfully parsed
+func LoggingCSRFCookiesInOAuthCallback(req *http.Request, cookieName string) {
+	cookies := req.Cookies()
+	if len(cookies) == 0 {
+		logger.Println(req, logger.AuthFailure, "No cookies were found in OAuth callback.")
+	} else {
+		var anyFound = false
```

Comment:
Leave some empty space above this to separate logical sections of the function
I'd also prefer if we could call this something more meaningful. Idiomatic to use `:=` to declare this as well, or `var anyFound bool`, but not `var anyFound = false`. Prefer the `:=`.
```suggestion
		foundCSRFCookie := false
```